vocab/credentials/v2/vocabulary.csv

```diff
@@ -17,9 +17,9 @@ property,credentialSchema,,,credential schema,,cred:VerifiableCredential,cred:Cr
 property,credentialStatus,,,credential status,,cred:VerifiableCredential,cred:CredentialStatus,,The value of the `credentialStatus` property MUST be an instance of Credential Status.
 property,credentialSubject,,,credential subject,,cred:VerifiableCredential,,,An entity about which claims are made.
 property,evidence,,,evidence,,cred:VerifiableCredential,cred:CredentialEvidence,,The value of the `evidence` property MUST be one or more Credential Evidence instances.
-property,expirationDate,,,expiration date,,cred:VerifiableCredential,xsd:dateTime,,The value of the `expirationDate` property MUST be a string value of an ISO8601 combined date and time string representing the date and time the credential ceases to be valid.
```

Comment:
Note that everything below here is going to conflict with https://github.com/w3c/vc-data-model/pull/968/files.